source/_integrations/schedule.markdown

```diff
@@ -135,48 +122,53 @@ schedule:
           default: {}
 {% endconfiguration %}
 
-### Attributes
+## Attributes
 
-A schedule entity's state exports attributes that can be useful in
-automations and templates.
+A schedule entity's state exports attributes that can be useful in automations and templates.
 
 | Attribute | Description |
 | ----- | ----- |
 | `next_event` | A datetime object containing the next time the schedule is going to change state. |
+| Additional data | The entries in **Additional data** / `data` of a schedule block when the respective block is active. |
 
-### Automation example
+## Automation example
+
+A schedule creates an on/off (schedule) sensor within the times set.
+Using the `light_schedule` example from above in an automation might look like this:
```

Comment:
This should briefly explain what the example below does. When the schedule is activated, it should turn on the lights. 

The light will only be turned on when the schedule is activated.